entry_types/scrolled/package/src/entryState/legalInfo.js

```diff
@@ -0,0 +1,81 @@
+import {useEntryMetadata} from "./metadata";
+import {useEntryState} from "./EntryStateProvider";
+import {getItems} from "../collections";
+import {useMemo} from "react";
+
+/**
+ * Returns a string (comma-separated list) of copyrights of
+ * all images used in the entry.
+ * If none of the images has a rights attribute configured,
+ * it falls back to the default file rights of the entry's account,
+ * otherwise returns an empty string
+ *
+ * @example
+ *
+ * const fileRights = useFileRights();
+ * fileRights // => "author of image 1, author of image 2"
+ */
```

Comment:
To make these docs show up in the correct category, you need to add items to the [scrolled `documentation.yml` file](https://github.com/codevise/pageflow/blob/30b53db563d2fc1743e932a33b5e6ece56e58903/entry_types/scrolled/package/documentation.yml#L11)